src/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const artistRouter = require('./routes/artist');
+const albumRouter = require('./routes/album');
+
+const app = express();
+
+app.use(express.json());
+
+app.use('/artists', artistRouter);
+
+app.use('/artists', albumRouter);
+
+app.use('/albums', albumRouter);
+
+app.get('/', (req, res) => {
```

Comment:
You could bunch these together since they are all related to the same thing:
```
app.use(express.json());
app.use('/artists', artistRouter);
app.use('/artists', albumRouter);
app.use('/albums', albumRouter);
app.get('/', (req, res) => {
```